cycles.js

```diff
@@ -0,0 +1,71 @@
+//Cycles, while, for. First task
+
+// for(i = 1; i <= 50; i++){
+//   console.log(i);
+//   document.open();
+//   document.write(`<p>${i}</p>`);
+//   document.close();
+// };
+
+//Cycles, while, for. Second task
+
+// for(i = 1; i <= 100; i++){
+//   console.log(i);
+//   document.open();
+//   document.write(`<p>${i}</p>`);
+//   document.close();
+// };
+
+//Cycles, while, for. Third task
+
+// for(i = 11; i <= 33; i++){
+//   console.log(i);
+//   document.open();
+//   document.write(`<p>${i}</p>`);
+//   document.close();
+// };
+
+//Cycles, while, for. Fifth task
+
+// for(i = 0; i <= 100; i++){
+//   if(i % 2 === 0){
```

Comment:
how can we write condition here without === 0?

Reply:
i = 0; i <= 100; i += 2

Reply:
i added it in my js file

Reply:
rewrite without === 0

Reply:
fixed